interfaces/inc/c_pal/socket_transport.h

```diff
@@ -36,6 +36,13 @@ MU_DEFINE_ENUM(SOCKET_SEND_RESULT, SOCKET_SEND_RESULT_VALUES)
 
 MU_DEFINE_ENUM(SOCKET_RECEIVE_RESULT, SOCKET_RECEIVE_RESULT_VALUES)
 
+#define SOCKET_ACCEPT_RESULT_VALUES \
+    SOCKET_ACCEPT_OK, \
+    SOCKET_ACCEPT_ERROR, \
+    SOCKET_ACCEPT_NO_SOCKET
```

Comment:
>SOCKET_ACCEPT_NO_SOCKET [](http://example.com/codeflow?start=4&length=23)

I am worried about the NO_SOCKET wording, people might take this as that they didn't provide a socket_transport_handle parameter.  I think it should be something more like 

SOCKET_ACCEPT_NO_CONNECTION or SOCKET_ACCEPT_TIMEDOUT